zebrad/tests/acceptance.rs

```diff
@@ -748,7 +749,7 @@ fn sync_large_checkpoints_mainnet() -> Result<()> {
 }
 
 // Todo: We had a `sync_large_checkpoints_testnet` here but it was removed because
```

Comment:
```suggestion
// TODO: We had a `sync_large_checkpoints_testnet` here but it was removed because
```
Nit-picky change to make the `TODO` automatically highlight in Vim :sweat_smile:

Reply:
Nice one, we have 2 more `Todo` in the codebase, i never found a good reason to standardize until now. They are both in `zebrad/tests/acceptance.rs`. Will it worth to change these in this PR?

Reply:
Fixed using `fastmod` in 724fbbe0